apps/climatemappedafrica/src/config/index.js

```diff
@@ -32,9 +32,7 @@ import MercyKaragi from "@/climatemappedafrica/assets/images/Mercy-Karagi.png";
 import Piero from "@/climatemappedafrica/assets/images/Piero.png";
 import screenshot1 from "@/climatemappedafrica/assets/images/Screenshot 2021-06-14 at 12.51.45@2x.png";
 import Tricia from "@/climatemappedafrica/assets/images/Tricia-Govindasamy.png";
-import desktopLogo from "@/climatemappedafrica/assets/logos/Component61.svg";
-import drawerLogo from "@/climatemappedafrica/assets/logos/Group 4188@2x-white.png";
```

Comment:
What was this `drawerLogo`?

Reply:
I is used for the mobile drawer menu

Reply:
... then why are you just deleting it @kelvinkipruto? Don't we need one?

Reply:
@kilemensi We don't need it since the current navigation does not contain links to other sections/pages. @m453h comment below show where it is used.

Reply:
So we don't need the drawer then @kelvinkipruto ?

Reply:
I see this has been marked as resolved @kelvinkipruto , did we remove the drawer altogether?

Reply:
@kilemensi Not yet. Let me remove it.